content/en/docs/tutorials/services/connect-applications-service.md

```diff
@@ -59,7 +59,7 @@ to make queries against both IPs. Note that the containers are *not* using port
 the node, nor are there any special NAT rules to route traffic to the pod. This means
 you can run multiple nginx pods on the same node all using the same `containerPort`,
 and access them from any other pod or node in your cluster using the assigned IP
-address for the Service. If you want to arrange for a specific port on the host
```

Comment:
existing one is correct

Reply:
Yes, this is "globally" correct. But locally in this paragraph we are speaking about pods yes, not services. We start to speak about services in the next chapter of the text.